src/core/Akka.Streams.Tests/Dsl/RestartSpec.cs

```diff
@@ -479,129 +485,123 @@ public void A_restart_with_backoff_flow_should_run_normally()
         [Fact]
         public void A_restart_with_backoff_flow_should_restart_on_cancellation()
         {
-            this.AssertAllStagesStopped(() =>
```

Comment:
What was the problem with `AssertAllStagesStopped`

Reply:
Not all specs are designed in a way that all stages are stopped at this point so the spec fails.